meme_utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import random
+import praw
+import logger
+
+LOG = logger.LOG
+
+reddit = praw.Reddit(client_id=os.getenv("CLIENT_ID"),
+                     client_secret=os.getenv("CLIENT_SECRET"),
+                     user_agent=os.getenv("USER_AGENT")
+                     )
+
+topic = 'meme'
+query = 'bonk OR horny OR hornyjail OR waifu'
```

Comment:
should we put it to env variable with default value? or maybe later

Reply:
later it will be configurable, yep